content/en/md2html-setup.md

```diff
@@ -43,13 +42,30 @@ files:
 - Appendix_L.md
 - Appendix_M.md
 ---
+# Detailed instructions see at https://markdown.standardshub.io/md2html-setup.
 ```
->Detailed instructions see at https://markdown.standardshub.io/md2html-setup.
 
-### New Properties:
-* `organizationName`:  represents the name of the organization.
-* `copyrightDate`:         the copyright date is defined by this property.
-* `logo`:                          this allows each document to have its own logo
-* `watermark`:               this property inserts a watermark inside of the PDF. Possible values are: "DRAFT",  "CANDIDATE", "HISTORIC", "DEPRECATED", "PRE-APPROVED", "APPROVED".
+### Properties
+* `title`
+    * Title of the document, as displayed on the front page
+* `status`
+    * Document status, e.g., `Draft`, `Candidate`, `Approved`
+* `version`
```

Comment:
@gocadimic I have added the deleted properties from Version V1.x. In this way, we can provide an update on each existing properties.

Reply:
@jpradocueva, 
I understand, in any case, we have that version for OMA.